lib/index.js

```diff
@@ -35,8 +35,8 @@ function getHint(str) {
 	return arr && arr[1];
 }
 
-function getTarFile(obj) {
-	return path.join(DIR, obj.site, obj.repo, `${obj.type}.tar.gz`);
+function getTarFile(obj, name) {
+	return path.join(name !== undefined ? path.join(HOME, '.' + name) : DIR, obj.site, obj.repo, `${obj.type}.tar.gz`);
```

Comment:
This could be:

```js
function getTarFile(obj, name) {
  let dir = name ? path.join(HOME, `.${name}`) : DIR;
  return path.join(dir, obj.site, obj.repo, `${obj.type}.tar.gz`);
}
```

Reply:
Easier to read IMO